src/jit/alloc.cpp

```diff
@@ -433,13 +427,12 @@ void ArenaAllocator::shutdown()
 }
 
 PooledAllocator PooledAllocator::s_pooledAllocator;
-LONG PooledAllocator::s_pooledAllocatorState = POOLED_ALLOCATOR_NOTINITIALIZED;
+LONG            PooledAllocator::s_pooledAllocatorState = POOLED_ALLOCATOR_NOTINITIALIZED;
```

Comment:
It's aligning it with the line above because we are using AlignConsecutiveDeclarations. It's weird, but it's what clang-format thinks it correct.